File: src/main/java/briscola/gui/model/FxmlPath.java
package briscola.gui.model;

public enum FxmlPath {
    MENU("Menu.fxml"),
    MATCH("Match.fxml");

    private final String path;

    FxmlPath(String path) {
        this.path = path;
    }

    public String getPath() {
        return path;
    }
}

File: src/main/kotlin/briscola/utils/FxmlPath.kt
package briscola.utils

/**
 * Enum class that contains the paths of the FXML files.
 * @param path the path of the FXML file.
 */
enum class FxmlPath(val path: String) {
    MENU("Menu.fxml"),
    MATCH("Match.fxml"),
    END_GAME("EndGame.fxml")
}